src/js/view/controls/components/button.js

```diff
@@ -19,11 +20,8 @@ export default function (icon, apiAction, ariaText, svgIcons) {
         });
     }
 
-    // Prevent button from being focused on mousedown so that the tooltips don't remain visible until
```

Comment:
I'm confused by the tooltip part of this comment. Are we sure the behavior is being respected ?